config/environments/development.rb

```diff
@@ -128,7 +128,7 @@
     Bullet.add_whitelist(type: :n_plus_one_query, class_name: "User", association: :profile)
 
     # Check if there are any data update scripts to run during startup
-    if %w[c console runner s server].include?(ENV["COMMAND"]) && DataUpdateScript.scripts_to_run?
+    if %w[Console Server].any? { |const| Rails.const_defined?(const) } && DataUpdateScript.scripts_to_run?
```

Comment:
Nice solution. Maybe we want to also include `DBConsole` here, which drops the user right into a Postgres shell.

Reply:
Did not know about that one, I will add it! Thanks!